src/styles/index.scss

```diff
@@ -1,3 +1,58 @@
+@import './utils/variables';
+@import 'block/header';
+@import 'block/pruduct-card';
+@import 'block/starts';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
According to the code style rules, there should be a space after `repeat` function: `repeat(4, 200px)` should be `repeat(4 , 200px)`.